app/schema/api/v1/models.rb

```diff
@@ -27,6 +27,10 @@ def medical_history
       }
     end
 
+    def appointment
+      super.tap { |d| d[:properties][:cancel_reason][:enum] -= %w(invalid_phone_number public_hospital_transfer moved_to_private) }
```

Comment:
This might read better using `merge`?

Reply:
I'd appreciate a code suggestion here actually. I agree this isn't elegant, since `tap` is generic. But I'd like to express "I'm removing some values from a nested hash value".